File: Data type - Conditions/M_Capital_or_Small_or_Digit.cpp
#include<bits/stdc++.h>
using namespace std;
#define endl '\n'
int main() {
    char a; cin >> a;
    if(int(a) >= 48 && int(a) <= 57) cout << "IS DIGIT";
    else if(int(a) >=  65 && int(a) <= 90) cout << "ALPHA" << endl << "IS CAPITAL";
    else if(int(a) >=  97 && int(a) <= 122) cout << "ALPHA" << endl << "IS SMALL";
    return 0;
}

File: Python/Conditions/M_Capital_or_Small_or_Digit.py
character = input()

if ord(character) >= 48 and ord(character) <= 64:
    print("IS DIGIT")
elif ord(character) >= 65 and ord(character) <= 96:
    print("ALPHA")
    print("IS CAPITAL")
else:
    print("ALPHA")
    print("IS SMALL")